javascript/js-parser/build.gradle

```diff
@@ -0,0 +1,9 @@
+plugins {
+    id 'kotlin'
+}
+
+
+dependencies {
+    //https://mvnrepository.com/artifact/org.graalvm.js/js
+    compile group: 'org.graalvm.js', name: 'js', version: gradle.graaljsVersion
```

Comment:
here's the way we actually add compile dependencies:
`implementation("org.jetbrains.kotlin:kotlin-stdlib:$gradle.kotlinVersion")`

Reply:
I changed that now